src/twitch-api.ts

```diff
@@ -73,6 +73,13 @@ class TwitchApi {
     return this.#broadcasterUser;
   }
 
+  private get moderator(): UserIdResolvable {
+    if (this.#botUserId == null) {
+      throw new Error("Tried to load chatters before client set up");
+    }
+    return this.#botUserId;
+  }
+
```

Comment:
We can remove this method as it is not used anywhere anymore and also the name is misleading.

Reply:
Done.